addons/l10n_ma/models/res_company.py

```diff
@@ -0,0 +1,22 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from markupsafe import Markup
+
+from odoo import api, fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    l10n_ma_ice = fields.Char("ICE", help='Common Company Identifier', default='')
+
+class BaseDocumentLayout(models.TransientModel):
+    _inherit = 'base.document.layout'
+
+    @api.model
+    def _default_company_details(self):
+        company_details = super()._default_company_details()
+        if self.env.company.country_code == 'MA':
+            company_details += Markup(f"<br> ICE: {self.env.company.l10n_ma_ice}")
```

Comment:
The argument of a `Markup` can only be a static string, otherwise you allow injections